docs/index.rst

```diff
@@ -169,6 +169,27 @@ You can also specify filters
 
     tasks = api.tasks.list(project=1)
 
+By defaults list returns all objects, eventually getting the
+paginated results behind the scenes.
+
+If you want to disable pagination support and get all the
+objects at once use the `all` argument
+
+.. code:: python
+
+   tasks = api.tasks.list(all=True)
+
+.. warning:: be aware that if the unpaginated results may exceed
+             the data the parser can handle and may result in an error.
+
+You can also explicitly control the pagination by passing `lazy_pagination`
+argument and the page number
+
+.. code:: python
+
+   tasks_page_1 = api.tasks.list(lazy_pagination)  # Will only return page 1
```

Comment:
Add example to get everything

Reply:
a plain `api.tasks.list()` will still return all the results, like it does now, but using the pagination to get all the results